docs/verifiable-document/dns-proof.md

```diff
@@ -4,8 +4,60 @@ title: Configuring DNS
 sidebar_label: Configuring DNS
 ---
 
-Overview
+Every OA document's provenance can be verified and traced back to it's creator or issuer. This is achieved by embedding an `identityProof` in the document which serves as a claim of identity. During the verification phase, the claim is checked against external records.
```

Comment:
It's creator => its creator

Claim of identity => claim for identity (not sure which one is best)